joladnijo/settings.py

```diff
@@ -28,6 +28,12 @@
 
 ALLOWED_HOSTS = ['*']
 
+CORS_ALLOWED_ORIGINS = [
+    'http://localhost:8000',
```

Comment:
Ezt elneztem elsore. Lokalban a frontend a 3000-es porton fog futni. A backend fut a 8000-es porton
```suggestion
    'http://localhost:3000',
```